clients/fides-js/src/components/Overlay.tsx

```diff
@@ -38,16 +34,6 @@ interface Props {
   renderModalContent: (props: RenderModalContent) => VNode;
 }
 
-/**
```

Comment:
I realized this wasn't needed anymore once I fixed how the updating cookie logic in place worked